.travis.yml

```diff
@@ -37,20 +37,26 @@ matrix:
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
       script:
-        - docker-compose run backend ./runtests.sh
+        - docker-compose run backend bash -c "pytest --cov=./ --cov-report=xml" ./runtests.sh
+      after_success :
+        - codecov -f coverage.xml -t 995256be-3a1f-4d10-b181-e24f88603b6b
 
     - env: python-tests-main
       language: minimal
       install:
         - docker-compose build
+        - pip install codecov --user
       script:
         # Several security features in settings.py (eg setting HSTS headers) are conditional on
         # 'https://' being in the site URL. In addition, we override the test environment's debug
         # value so the tests pass. The real environment variable will be checked during deployment.
         - docker-compose run -e SITE_URL=https://treeherder.dev -e TREEHERDER_DEBUG=False backend python -bb ./manage.py check --deploy --fail-level WARNING
         # Using `-bb` mode to surface BytesWarnings: https://docs.python.org/3.7/using/cmdline.html#cmdoption-b
-        - docker-compose run backend python -bb -m pytest tests/ --runslow --ignore=tests/selenium/
+        - sudo docker-compose run backend bash -c "pytest --cov=./ --cov-report=xml" python -bb -m pytests tests/ --runslow --ignore=tests/selenium
```

Comment:
Change it to:
```bash
docker-compose run backend bash -c "pytest --cov-report=xml tests/ --runslow --ignore=tests/selenium"
```
Notice that I've changed the 2nd quote, I've removed "sudo" and dropped the `-bb` (this change is unrelated to your patch; it was simply overdue).

Reply:
Okay. Will make the required changes

Reply:
If I am applying ```" "``` across the entire statement, it's throwing me the file not found error. That's why I wrote it as ```docker-compose run backend bash -c "pytest --cov-report=xml" tests/ --runslow --ignore=tests/selenium```
![Screenshot from 2020-01-31 23-22-49](https://user-images.githubusercontent.com/29003047/73562251-bfa53780-4480-11ea-9fdf-d9e0fc33ea8b.png)